### DISCUSSION TITLE
glTF conformance

### DISCUSSION THREAD

**Discussion by Aloalo** [google/model-viewer#2016] [General]
The new glTF sample vewier launched recently (http://gltf.ux3d.io/), and it seems to be fully conforming to the glTF 2 specification. This includes multiple UV sets, different texture transforms on a single material, as well as all ratified extensions. Is the model-viewer aiming for the same level of conformance to the glTF standard in the future?

**Comment by elalish** [comment]
Ideally, though our focus is more on performance and reliability of rendering across devices than the sample viewer is. Our glTF conformance comes from three.js, so fixing all the edge cases comes with contributions there. However, we've found that workarounds for mobile GPU driver bugs have benefited a lot more of our customers than supporting an extra set of UVs would so far.
